clouddriver-core/clouddriver-core.gradle

```diff
@@ -1,4 +1,6 @@
 dependencies {
+  api("com.netflix.spinnaker.kork:kork-plugins")
```

Comment:
should this be api (vs implementation)

where else in the code is stuff from this lib referenced

Reply:
I copied what was done in orca